syntax = "proto3";

package proto;

service AService {
    rpc TestVerificationCode (TestVerificationCodeRequest) returns (TestVerificationCodeResponse);

    rpc IntegralChange (IntegralChangeRequest) returns (IntegralChangeResponse);
}

message TestVerificationCodeRequest {
    string phone = 1;
}

message TestVerificationCodeResponse {
    string username = 1;
    string phone = 2;
    string email = 3;
}

message IntegralChangeRequest {
    string order_number = 1;
}

message IntegralChangeResponse {
    bool success = 1;
    string message = 2;
}